audit/event_formatter.go

```diff
@@ -0,0 +1,571 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package audit
```

Comment:
This file contains an eventlogger.Node implementation that is a Formatter Node. Because the JSON and JSONX formats are so similar, we've implemented them both using the same type. JSONX is XML and its encoder takes JSON as the input.